helm/cert-manager-app/charts/giantswarm-cluster-issuer/Chart.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: 1.0.0
```

```suggestion
appVersion: 0.9.0
```

I read in values.yaml `tag: v0.9.0`